WORKSPACE

```diff
@@ -0,0 +1,64 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_apple",
+    sha256 = "0052d452af7742c8f3a4e0929763388a66403de363775db7e90adecb2ba4944b",
+    url = "https://github.com/bazelbuild/rules_apple/releases/download/0.31.3/rules_apple.0.31.3.tar.gz",
```

Comment:
Nit: 0.32.0 is now available, can you update to that? https://github.com/bazelbuild/rules_apple/releases/tag/0.32.0